charts/tekton-pipelines/values.yaml

```diff
@@ -60,13 +60,19 @@ buildpacks:
           resources: {}
           workingDir: $(resources.inputs.app.path)
           script: |
-            #!/bin/bash
-            [ -f package.json ] && (
-              echo "install node.js dependencies"
-              yes | npm install --silent;
-              npm run build:$(params.environment);
```

Comment:
Because of this `;` in the end of `npm run build:$(params.environment);` bash procceeds to next statement anyway, regardless there was error on build or not. Because next statement succeeds, build error is swallowed and deploy of incomplete build result continues.